vector/src/main/res/xml/vector_settings_general.xml

```diff
@@ -22,33 +22,11 @@
             android:summary="@string/change_password_summary"
             android:title="@string/settings_password" />
 
-        <!-- Email will be added here -->
-
-        <!-- Note: inputType does not work, it is set also in code, as well as iconTint -->
-        <im.vector.app.core.preference.VectorEditTextPreference
-            android:icon="@drawable/ic_material_add"
-            android:inputType="textEmailAddress"
-            android:key="ADD_EMAIL_PREFERENCE_KEY"
-            android:order="100"
-            android:title="@string/settings_add_email_address"
-            app:iconTint="@color/riotx_accent" />
-
-        <!-- Phone will be added here -->
-
-        <!-- Note: iconTint does not work, it is also done in code -->
         <im.vector.app.core.preference.VectorPreference
-            android:icon="@drawable/ic_material_add"
-            android:key="ADD_PHONE_NUMBER_PREFERENCE_KEY"
-            android:order="200"
-            android:title="@string/settings_add_phone_number"
-            app:iconTint="@color/riotx_accent" />
-
-        <im.vector.app.core.preference.VectorPreference
```

Comment:
This setting has been removed by mistake, please restore

Reply:
Nice catch!